Makefile

```diff
@@ -17,4 +17,10 @@ ssh:
 	(cd template; vagrant ssh)
 
 destroy:
-	(cd template; vagrant destroy -f)
\ No newline at end of file
+	(cd template; vagrant destroy -f)
+
+# submodules
+# https://www.vogella.com/tutorials/GitSubmodules/article.html
+## Point to latest commit in remote submodule
+update-template:
```

Comment:
It will be nice to add some info about this command in the README docs(if not already done):
1. Purpose of this command
2. When should this command be executed
3. Any other important notes?

Reply:
Hm, the reason I didn't add this in the first place is because the user should never run this, it's only used by the other targets. Should there still be a doc for it?